pkg/virt-api/rest/authorizer.go

```diff
@@ -196,7 +196,7 @@ func isInfoEndpoint(req *restful.Request) bool {
 	// /apis/subresources.kubevirt.io/v1alpha1/namespaces/default/virtualmachines/testvm/console
 	// The /apis/<group>/<version> part of the urls should be accessible without needing authorization
 	pathSplit := strings.Split(httpRequest.URL.Path, "/")
-	if len(pathSplit) <= 4 {
+	if len(pathSplit) <= 4 || (len(pathSplit) > 4 && pathSplit[4] == "version") {
```

Comment:
we shouldn't have to make an exception here. Version likely doesn't belong as the 4th arg there.

Reply:
As I say before there is some difficulties to add it just as /version